docs/src/pages/system/properties/properties.md

```diff
@@ -1,19 +1,30 @@
 # Properties
 
-<p class="description">This page lists all of the custom system properties, how are they linked with the theme, and which CSS properties they compute.</p>
+<p class="description">This page lists all of the custom system properties, how are they linked with the theme, and which CSS properties they compute. All other regular CSS properties and selectors are supported too.</p>
```

Comment:
Forgive me if I'm missing something obvious (not a first!), but in what sense are _all_ are CSS selectors supported?

Reply:
You can specify any valid CSS as javascript object, which includes use of pseudo selectors, like :hover, or class selectors, like & .MuiButton-root: {} etc

Reply:
Yes, I was thinking that "all" might be a bit broad, but I guess trying to caveat it would make it overly complicated.